articles/understanding-differential-privacy/index.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/understanding-differential-privacy/
+title: Understanding Differential Privacy
+description: Introduction to differential privacy and methods used to preserve privacy in Databases
+author: srishilesh-p-s
+date: 2020-10-06T00:00:00-12:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/understanding-differential-privacy/hero.jpg
+    alt: Understanding Differential Privacy
+---
+
+In this article, we will get an overview of resolving the privacy issue using Differential privacy. You will understand the basics of how privacy is preserved in Databases used for Machine learning and Deep learning. Though understanding differential privacy requires a Mathematics background, this article covers a very basic overview of the concepts. You will also be coding a sample database and check if it is differentially private or not.
+<!--more-->
+
+### Table of contents
+
+- [Introduction](#introduction)
+- [Differential Privacy](#differential-privacy)
+- [Differencing attacks](#differencing-attacks)
+- [Implementing differential privacy](#implementing-differential-privacy)
+- [Types of Differential privacy](#types-of-differential-privacy)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+### Introduction
+
+In this digital era, most of the companies are data-driven. Our data is harvested to keep growing their economic incentives, in exchange for services provided to us. Research institutions often use and share data containing confidential information about individuals. Improper disclosure of such data can have adverse consequences for a data subject’s private information, or even lead to civil liability or bodily harm.
+
+In the late 2000s, Netflix ran a competition for building a recommendation system. To build, they released a large dataset by anonymizing the data which didn’t leak the movie names or users' ratings. Two professors from the University of Texas [de-anonymized](https://www.cs.utexas.edu/~shmat/shmat_oak08netflix.pdf) them completely by comparing them with IMDB ratings, thus revealing the personal information about the dataset. Data privacy is compromised! Similarly, one such data breach of health records happened in 1997 too.
+
+This sort of data leakage is very alarming both to the users as well as organizations. To resolve this, we can either completely randomize the way the data is sent, using keys that are only decipherable at either end (Differential Learning). Well, there is an alternative to use Federated learning, where we bring the model to the data instead of vice-versa. Fundamentally, we must understand the concept of Differential Privacy to check for leakage in data.
```

Comment:
(Differential Learning). Well, there is an alternative ->  (Differential Learning). There is an alternative